<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="networkField">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QListWidget" name="rangeField">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>181</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QListWidget" name="nameField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>88</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
